pkg/policy/l4.go

```diff
@@ -337,17 +346,27 @@ func (l4 *L4Policy) GetModel() *models.L4Policy {
 	}
 
 	ingress := []string{}
-	for _, v := range l4.Ingress {
+	for _, v := range l4.Ingress.Filters {
 		ingress = append(ingress, v.MarshalIndent())
 	}
+	ingressLabels := make([][]string, 0, len(l4.Ingress.SourceRuleLabels))
+	for _, lbls := range l4.Ingress.SourceRuleLabels {
+		ingressLabels = append(ingressLabels, lbls.GetModel())
+	}
 
 	egress := []string{}
-	for _, v := range l4.Egress {
+	for _, v := range l4.Egress.Filters {
 		egress = append(egress, v.MarshalIndent())
 	}
+	egressLabels := make([][]string, 0, len(l4.Egress.SourceRuleLabels))
+	for _, lbls := range l4.Egress.SourceRuleLabels {
+		egressLabels = append(egressLabels, lbls.GetModel())
+	}
 
 	return &models.L4Policy{
-		Ingress: ingress,
-		Egress:  egress,
+		Ingress:                 ingress,
+		IngressSourceRuleLabels: ingressLabels,
```

Comment:
`IngressSourceRuleLabels` and `EgressSourceRuleLabels ` . I am not sure we need `Source`   in the variable name. Seems confusing?

Reply:
I'm afraid these names have to match the name in the API json, mangled by whatever swagger does (which seems to snake-case it and remove the dashes). I will be changing them to be derived-from-rules in the api, so they would look like `DerivedFromRules` which is a little closer to what you suggested?